roadmap.md

```diff
@@ -0,0 +1,66 @@
+# <img src="https://opentelemetry.io/img/logos/opentelemetry-logo-nav.png" alt="OpenTelemetry Icon" width="45" height=""> OpenTelemetry Project Roadmap
+
+OpenTelemetry is a healthy open source community, and contributors are free to propose new workstreams or work on any part of the project that they desire. However, there’s also value in the project focusing community members of specific bodies of work and releases, as this allows us to form a more cohesive set of capabilities (unified semantics, single implementations for each language, etc.) and to ship the project in a more impactful way (new signal types release across multiple languages at once, etc.).
+
+This roadmap is not _law_, and it is not meant to be used to force people to work on specific projects - this is an open source project and we (community members, the governance committee, etc.) are not anyone’s boss. Rather, it exists to provide direction to new contributors, to the public and end users who want to know what’s coming next, and to attempt to channel the bulk of our concentration and development efforts into the areas where they are most needed.
+
+# What We’ve Accomplished
+OpenTelemetry was started in 2019 with the promise of making it easy and consistent for developers to capture distributed traces and metrics from their applications and infrastructure using SDKs, the Collector, and OTLP. Since then, we've delivered:
+
+- A specification for distributed traces and metrics, which defines the objects used to represent each, how to interact with them, and the types of data that are expected for different sources.
+- Semantic conventions for metadata, which define how resources and other components are represented. This is applied consistently across all signal types, which allows traces, metrics, and other signals to be processed consistently and correlated.
+- SDKs for twelve languages that allow developers to capture telemetry from their services and to create their own custom telemetry.
+- Automatic instrumentation agents for __ languages, which allow anyone to capture telemetry from their services without making code changes or redeploying.
+- Instrumentation libraries for thousands of pieces of software, including operating systems, container runtimes, databases, language runtimes, and libraries, all of which allow the Collector (for infrastructure and third-party applications), and SDKs and automatic instrumentation agents (for custom applications) to capture signals, capture metadata, and propagate context automatically.
+- A protocol, OTLP, for transmitting telemetry and metadata between OpenTelemetry components and to backends for processing.
+
+# Major Priorities
+All of the following workstreams are major areas of investment for OpenTelemetry: all of them have large groups of people already focused on them. Their priorities primarily reflect the ordering of when we expect them to be released.
```

Comment:
```suggestion
All of the following workstreams are major areas of investment for OpenTelemetry: all of them have large groups of people already focused on them. Their priorities primarily reflect the ordering of when we expect them to be released.

These priorities are most useful when comparing them against each other. For example, P0 entries are more critical than P1 entries, which are in turn more critical than P2s. We do have some basic definitions of each priority level, however:

- P0: We must do this within the specified timeframe (if there is one), or we will fail as a project
- P1: This is one of our most important major initiatives, and it will have a major impact on the project
- P2: This work is important enough to be tracked and prioritized and is more important than items not on this list, but it is currently lower urgency or criticality than P0 and P1 entries
```